DeviceManager/main.py

```diff
@@ -9,13 +9,10 @@
 import DeviceManager.ErrorManager
 
 from .DatabaseHandler import db
-from .StatusMonitor import StatusMonitor
 from .TenancyManager import list_tenants
 
 with app.app_context():
     g.tenant = '__status_monitor__'
```

Comment:
Why have this remained ?

Reply:
It shouldn't.

Reply:
Sorry, misread your comment. Actually this line has been removed.